presubmit.py

```diff
@@ -260,17 +260,30 @@ def license_check(paths: List[Path]) -> bool:
 
 def get_changed_files() -> List[Path]:
     """Return a list of absolute paths of files changed in this git branch."""
-    diff_command = ['git', 'diff', '--name-only', 'FETCH_HEAD']
+    uncommitted_diff_command = ['git', 'diff', '--name-only', 'HEAD']
```

Comment:
I dont think this is needed, only origin... worked for me.

Reply:
Even for files that you've changed but not modified?
I think that is why this was originally reverted.

Reply:
this was reverted because it failed in CI (something with origin not existing).

Reply:
Ah, fixed.